### PR TITLE
RF-2789 Return false exit status for unknown flags

### PR DESCRIPTION
**Pull request by jbarber**
So that giving bad flags doesn't appear the same as having a passing
test suite

https://rainforest.atlassian.net/browse/RF-2789

No idea how to test this?

```
$ ./rainforest-cli --foo
2017/08/26 00:19:29 Unknown argument
```

### CODE REVIEW

**Comment by epaulet** [comment]
While we're at it, why not also add the `CommandNotFound` function? Or is there already a reasonable default?

**Comment by jbarber** [comment]
@epaulet 
```
$ ~/usr/bin/rainforest asdasd
2017/08/29 12:34:04 No help topic for 'asdasd'
$ echo $?
3
```

So I think it's handled already?

**Comment by jbarber** [comment]
@epaulet is there anything else that needs to be done to release this apart from merging?

**Comment by epaulet** [comment]
After we merge, we will need to update the version constant and the change log. Once all that's done in and in the master branch, we just need to tag the final commit with the version number. That will trigger the circle build that releases the new version. Then we just need to update the Github change logs.

File: README.md [hunk 1]
```diff
@@ -288,3 +288,29 @@ Email [help@rainforestqa.com](mailto:help@rainforestqa.com) if you're having tro
 4. Commit your changes (`git commit -am 'Add some feature'`)
 5. Push to the branch (`git push origin my-new-feature`)
 6. Create new Pull Request
+
+## Release process
+
+Check the `circle.yml` for the lastest, but currently merging to master will build and deploy to the following Equinox channels:
```
Comment (epaulet) [inline README.md]: Typo on `latest`

**Comment by jbarber** [comment]
@epaulet can you re-review the changes I've made here. The CHANGELOG reformating is to make it more consistent and make it easier to automatically update the GH releases page (see #292)

**Pull request merged**